betatest/admin.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
```

Comment:
@smarnach Are you sure trolling about the enforceability of the GPL over the format of headers is a good use of our time here? I'd be happy to chat about the FSF and the GPL around a beer though, and I will definitely be interested in knowing which parts of the GPL are unenforceable - once you sprint is clean can you send me the details?

Also, besides the technical legal skills, there are other elements to consider when considering legal advice, such as the motivations of the organization giving them, and how this might fit their larger strategy. I don't always agree with everything the FSF does or say, but at least their motivations are clear and strongly held to. They align in many ways with ours. On the other side, on licensing Google is quite opposed to many of the principles of free software - cf the fact that the vast majority of the code they produce is proprietary, and their position on software patents (which is behind a lot of their opposition to the GPLv3, btw).